src/test.zig

```diff
@@ -611,6 +636,22 @@ pub const TestContext = struct {
             }
 
             switch (update.case) {
+                .Header => |expected_output| {
+                    var file = try tmp.dir.openFile("test_case.h", .{ .read = true });
+                    defer file.close();
+                    var out = file.reader().readAllAlloc(arena, 1024 * 1024) catch @panic("Unable to read headeroutput!");
+
+                    if (expected_output.len != out.len) {
+                        std.debug.print("\nTransformed header length differs:\n================\nExpected:\n================\n{}\n================\nFound:\n================\n{}\n================\nTest failed.\n", .{ expected_output, out });
+                        std.process.exit(1);
+                    }
+                    for (expected_output) |e, i| {
+                        if (out[i] != e) {
+                            std.debug.print("\nTransformed header differs:\n================\nExpected:\n================\n{}\n================\nFound:\n================\n{}\n================\nTest failed.\n", .{ expected_output, out });
+                            std.process.exit(1);
+                        }
+                    }
```

Comment:
These should use `std.testing.expectEqualStrings`.

Reply:
As @pixelherodev mentioned, it's done like this elsewhere in `test.zig`. I'm planning to follow up with a PR after this to make this change across the file.